src/components/networked.js

```diff
@@ -330,95 +315,42 @@ AFRAME.registerComponent('networked', {
           if (childEl) { // Is false when first called in init
             if (schema.property) {
               childEl.setAttribute(schema.component, schema.property, data);
-            }
-            else {
-              var shouldLerp = this.shouldLerpComponent(syncedComponents, schema);
-              this.updateComponent(childEl, schema.component, data, shouldLerp);
```

Comment:
Did we drop support for toggling lerp per component?

Reply:
There is no longer a lerp component. Lerping is always enabled now for components defined in network-schemas, unless lerping is disabled globally. Do you think it is important to be able to disable lerp on a per component basis? Networked-aframe master currently doesn't allow it either, iirc, but that could conceivably be added in.

Reply:
I think we wanted to disable lerping on the screen sharing screen, for example. But I was mostly just surprised that this PR removed this ability.

Reply:
We needed to do that because the old lerping style didnt really handle "sparse" updates correctly. So when you tried to move a shared screen it would lerp to the new position over the duration since the last time it was moved. That is not an issue with this update.

Reply:
Ah, fair enough.